addons/website/static/src/builder/plugins/options/card_image_position_option.js

```diff
@@ -0,0 +1,29 @@
+import { BaseOptionComponent, useDomState } from "@html_builder/core/utils";
+
+export class CardImagePositionOption extends BaseOptionComponent {
+    static template = "website.CardImagePositionOption";
+    static props = {
+        label: { type: String },
+        level: { type: Number, optional: true },
+    };
+    static defaultProps = {
+        level: 0,
+    };
+
+    setup() {
+        super.setup();
+        this.state = useDomState(async (editingElement) => {
+            const imageEl = editingElement.querySelector(".o_card_img");
+            if (!imageEl || "shape" in imageEl.dataset) {
+                return { show: false };
+            }
+            if (!imageEl.complete) {
```

Comment:
Just to be sure: in `CardImageAlignmentOption `, there was a condition on `hasCoverImage`, can we remove it here?

Reply:
`hasCoverImage` checked if there is an `.o_card_img_wrapper` element. But from what I have seen in the code, `.o_card_img` is always inside a `.o_card_img_wrapper` (and conversely) so the condition is still checked here